File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.cc
#include <iostream>
using namespace std;

int main() {

	int n;

	cin >> n;

	int num[1001];

	int array[1001];
	array[0] = -1001;


	for (int i = 1;i <= n;i++) {
		cin >> num[i-1];
		array[i] = 1001;
	}

	for (int y = 1;y <= n;y++) {
		for (int i = 0;i < n;i++) {
			if (num[i] < array[y]&& num[i] > array[y-1]) {
				array[y] = num[i];
			}
			
		}
	}

	for (int i = 1;i <= n;i++) {
		cout << array[i]<<endl;
	}
}

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int s = Integer.parseInt(br.readLine());
        ArrayList<Integer> list = new ArrayList<>();
        for (int i = 0; i < s; i++) {
            int num = Integer.parseInt(br.readLine());
            list.add(num);
        }
        Collections.sort(list);
        for (int i = 0; i < list.size(); i++) {
            bw.write(list.get(i)+"\n");
        }
        bw.flush();
        bw.close();
    }
}

